src/Tests/Logic/Forms/RemoveTextForHearImpairedTest.cs

```diff
@@ -1322,7 +1322,7 @@ public void RemoveTextForHiInDialogue1()
         public void RemoveTextForHiInDialogue2()
         {
             var target = GetRemoveTextForHiLib();
-            target.Settings.RemoveTextBetweenBrackets = true;
+            target.Settings.RemoveTextBetweenSquares = true;
             string text = "-[Ronnie laughs]" + Environment.NewLine + "-[sighs] Wow, Ronnie.";
```

Comment:
The mistake can clearly be noticed here... we are expecting to remove `[` but the settings was configured to `{`... why are the test passing? 😕